pkg/plugin/plugin.go

```diff
@@ -121,14 +121,30 @@ func makeAutoscaleEnforcerPlugin(ctx context.Context, obj runtime.Object, h fram
 	submitVMBoundsChanged := func(vm *api.VmInfo, podName string) {
 		pushToQueue(func() { p.handleUpdatedScalingBounds(vm, podName) })
 	}
+	submitNonAutoscalingBoundsChanged := func(oldVM *api.VmInfo, newVM *api.VmInfo, name util.NamespacedName) {
+		pushToQueue(func() {
+			vmname := newVM.Name
+			cpuDiff := newVM.Using().VCPU - oldVM.Using().VCPU
+			memDiff := newVM.Using().Mem - oldVM.Using().Mem
+
+			p.state.lock.Lock()
+			defer p.state.lock.Unlock()
+
+			p.state.nodeMap[vmname].memSlots.Reserved += memDiff
+			p.state.nodeMap[vmname].vCPU.Reserved += cpuDiff
+
+			p.state.podMap[name].memSlots.Reserved += memDiff
+			p.state.podMap[name].vCPU.Reserved += cpuDiff
```

Comment:
I opted to just leaved everything the handler without making a shim because it's really only setting stuff - there's no other calculation. It seems like use cases like this are rare when looking at the other "actual" handlers, which all perform some calculation.

Reply:
Sure, makes sense